src/index.html

```diff
@@ -10,19 +10,25 @@
         <div id="the-game"></div>
         <div class="settings">
             <nav class="mobile-nav">
-                navigation and menu link
-                <button class="mobile-nav-button">menu</button>
+                <button class="mobile-nav-button">Menu</button>
             </nav>
             <aside class="profile-wrapper">
                 <h2>The game</h2>
                 <div>
-                    stats
+                    Stats
                 </div>
                 <div>
-                    equipment
+                    Equipment
                 </div>
                 <div>
```

Comment:
you can add here class eg. 'players-list' and keep it as independent as possible with having children elements following BEM convention, eg. players-list__header, players_list__count